### ISSUE TITLE
BWL Packs + Boss

### ISSUE THREAD

**Issue by keaarori** [LightsHope/server#882]
Currently there is a Death Talon Overseer after Broodlord and there should not be one. 

Both of these videos you can clearly see through the gate and see it does not exist.
https://www.youtube.com/watch?v=SwhMTVZkOag (0:34)
https://www.youtube.com/watch?v=5F4dpn8xJvQ (5:45)

Beginning of this video - No Overseer
https://www.youtube.com/watch?v=Aci8pBVpUCA

Also the recent change to positioning on Wyrmgard is incorrect 
https://www.youtube.com/watch?v=Aci8pBVpUCA at 0:10 you can clearly see exactly how its supposed to be. (Overseer and Wyrmguard are mixed up)
At 0:17-18 you can see that overseer is closer to the far wall behind the warlock for his pat.

At 0:23 it appears that one should be further to the left closer to the spellbinder

Other than that it appears that boss should not pull the packs around it while running. (Rest of video)

**Comment by GuybrushGit** [comment]
Good videos all this seems correct, but about the change in the Wyrmguard positions, it was made based on this video: https://www.youtube.com/watch?v=dPI3y0Kjcas&feature=youtu.be&t=280
All the positions are the same except the overseer and wyrmguard are switched around. It could be that their position is randomized.

**Comment by bgfds** [comment]
Why do you care about the difficulty of these trash packs or their positioning? You skip them all anyways every single week.

**Comment by keaarori** [comment]
Github is not the place to attempt to bash other people, I saw it was off while watching a video and as its not blizzlike Ill have it changed.

**Comment by bgfds** [comment]
Hahah yea, you were just doing a bit of light youtube watching, looking at videos titled "Exploit Bug BWL" and you just happened to notice minute positioning discrepancies that would make skipping these packs easier for your elite guild

**Comment by bigfgeg** [comment]
I agree bgfds let's make custom changes to the server that prevent blizzlike pathing from being used
